content/src/cloud/faq-subscription-payments/faq-attach-vat-number.md

```diff
@@ -6,8 +6,14 @@ zendesk:
   labels: cloud, billing, pricing
 ---
 
-We strictly operate as a business-to-consumer provider, as such we can not attach your VAT number to the invoice.
+We strictly operate as a business-to-consumer provider. As such, we cannot include your VAT number on the invoice.
+
+If you’re located in the EU or UK, and you’re seeing VAT charged, it’s because our services are provided to private individuals. We are VAT registered where required by law, but as a business-to-consumer provider, our VAT registration does not allow customers to reclaim or reverse-charge VAT.
```

Comment:
> If you’re located in the EU or UK, and you’re seeing VAT charged

That makes it sound optional?
But it applies to everyone in EU/UK right?

Reply:
<https://github.com/NabuCasa/support/pull/423>